readme.md

```diff
@@ -22,8 +22,8 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://RiTAnd.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://RiTAnd.github.io/layout_product-cards/report/html_report/)
```

Comment:
you need to fix your demo links, try to run the `npm run deploy`, or feel free to ask for some help in the chat